src/components/Picklist/index.js

```diff
@@ -129,10 +131,15 @@ class Picklist extends Component {
 
     getValue() {
         const { value } = this.props;
-        if (value && typeof value === 'object') {
-            return value;
+        return normalizeValue(value);
+    }
```

Comment:
don't need to use getValue instead call normalizeValue directly where it is used
also change the helper name to getNormalizeValue